client/view/workbook/info/info.html

```diff
@@ -0,0 +1,50 @@
+<div class="container">
+
+
+    <header>
+        <h2>{workbook.name}</h2>
+    </header>
+
+
+    <main>
+        {#if forgotten}
+            {workbook.name} has been deleted.<br>
+            <button style="background-color: var(--green);" on:click="restoreWorkbook({key}, {workbook})">Restore</button>
```

Comment:
The curly braces are unnecessary here, they're being interpreted as the object literal syntax, which is why you're getting an object passed in to the `restoreWorkbook` method below.
```suggestion
            <button style="background-color: var(--green);" on:click="restoreWorkbook(key, workbook)">Restore</button>
```

If you drop the object literal syntax and just pass in values, the methods will be a lot easier to read.